_pages/timer1isr.md

```diff
@@ -5,4 +5,9 @@ permalink: /timer-1-isr/
 exclude: true
 ---
 
-![Int0isr]({{'/puml/svgs/DetailedDesign-Timer1isr.svg' | relative_url}})
+Triggered every **41.76ns**
```

Comment:
Timer 1 is configured to be triggered each 1 communication bit at a fixed rate, and synchronized every time int0 detects a falling edge. 

Int0 detects falling DSHOT rx edges and stores time between falling edges in Temp1 pointer used in DecodeShot2Bit.puml. Temp1 pointer is set at address 0 of XRAM, so XRAM is used to store those edges.